bin/build-db

```diff
@@ -6,6 +6,10 @@
  * is impossible in an Elm test environment.
  */
 const fs = require("fs");
+const textileProcesses = process.env.TEXTILE_PROCESSES || "public/data/textile/processes.json";
+const foodProcesses = process.env.FOOD_PROCESSES || "public/data/food/processes.json";
```

Comment:
What's the use case for using different output targets? Debugging?

Reply:
I'm not sure what you mean by that, we store the food and textile processes in their own files, because they don't have the same structure, and it wouldn't make much sense to merge them both in the same file and use a `Decode.oneOf`.

Am I missing something?

Reply:
We need to output EVERYTHING in those files for the test run (all the impacts), but not for the normal build ;)